src/api/vault/vault.ts

```diff
@@ -85,3 +90,43 @@ export const fetchVaultExecuteQuery: QueryFunction<IADSApiSearchResponse['respon
   const { data } = await api.request<IADSApiSearchResponse['response']>(config);
   return data;
 };
+
+/** user data request **/
+export const useGetUserData: ADSQuery<unknown, IADSApiUserDataResponse> = (_, options) => {
+  return useQuery({
+    queryKey: 'user-data',
```

Comment:
Should add `user-data` to the vaultkeys map above.  Makes it simpler for fetching server-side